generatedocs.py

```diff
@@ -81,10 +116,32 @@ def get_lines_and_content_from_file(asset_file, regex, look_for_header = False):
 
       # If we are in header comment, split into header and description
       if look_for_header and is_header_comment:
-        comment = line.split(LUA_COMMENT)[1]
+        # Remove the beginning of the line that marks it being a Lua comment
+        comment = line[len(LUA_COMMENT):]
+        if comment[0] == " ":
+          comment = comment[1:]
+
         if l_no == 1:
+          # The first line becomes the header
           header = comment
         else:
+          if comment.startswith(LITERAL_INCLUDE):
+            # Literal includes have to be handled a bit separately. While writing examples
+            # we want to pretend that the file we are including is relative to the Lua
+            # file, but for Sphinx, it is relative to the Markdown file. To solve this, we
+            # make use of the fact that Sphinx will treat any path to a literalinclude
+            # that starts with a / as a path relative to the _base folder_. So we can
+            # switch out the path at the last minute and make sure that it gets correctly
+            # included
+            print("Before", comment)
```

Comment:
There needs to be a new line before this "Before". It prints out on the last line now making this section a bit hard to read

Reply:
I forgot to remove those debug prints in here actually